views/active-directors.html

```diff
@@ -36,14 +36,11 @@
       </span>
       <h1 class="govuk-heading-l">Current directors</h1>
 
-        {{ govukButton({
-          attributes: {
-            id: "submit",
-            style: appointDisabled,
-            "data-event-id": "appoint-new-director-button"
-          },
-          text: "Appoint new director"
-        }) }}
+      {% if ap01Active %}
+        <button id="submit" class="govuk-button" onclick="_paq.push(['trackGoal', '{{PIWIK_APPOINT_DIRECTOR_START_GOAL_ID}}'])" data-event-id="remove-director-button" data-module="govuk-button">
```

Comment:
is the `data-event-id` correct?  is says `remove-director-button`, should it be `appoint-director-button`?